e2e/app/page/authenticated-page.ts

```diff
@@ -40,8 +39,8 @@ export default abstract class AuthenticatedPage extends BasePage {
   /**
    * Load AoU page URL.
    */
-  async loadPageUrl(url: PageUrl): Promise<void> {
```

Comment:
why change this?  this looks like a *decrease* in typesafety to me.

Reply:
I don't think it added any real benefit.
Updated to new signature `gotoUrl(url: { value: string })` to enforce type of properties in structured object.

```
export const PageUrl = {
  Home: { value: LOGIN_URL },
  Workspaces: { value: LOGIN_URL + WORKSPACES_URL_PATH },
  Admin: { value: LOGIN_URL + ADMIN_URL_PATH },
  Profile: { value: LOGIN_URL + PROFILE_URL_PATH }
} as const;
```

Reply:
ok, I don't feel strongly about this

Reply:
I don't understand why that's an improvement on using `PageUrl` or simply using strings

Reply:
What do you think about this idea? https://github.com/all-of-us/workbench/pull/5232/files

Reply:
I like it. Can I copy changes into this PR?

Reply:
Cool!  Please click merge on my PR, and it will copy into here automatically.

This is the magic of Stacked PRs!

Reply:
This is new to me. Thanks for the great github tips.